SUPPORTED_APIS.md

```diff
@@ -2,31 +2,122 @@
+| `ETH` | `eth_accounts` | NOT IMPLEMENTED | Returns a list of addresses owned by client |
+| `ETH` | `eth_blockNumber` | NOT IMPLEMENTED | Returns the number of the most recent block |
+| `ETH` | `eth_call` | NOT IMPLEMENTED | Executes a new message call immediately without creating a transaction on the block chain |
+| `ETH` | `eth_chainId` | NOT IMPLEMENTED | Returns the currently configured chain id |
+| `ETH` | `eth_coinbase` | NOT IMPLEMENTED | Returns the client coinbase address |
+| `ETH` | `eth_estimateGas` | NOT IMPLEMENTED | Generates and returns an estimate of how much gas is necessary for the transaction to complete |
+| `ETH` | `eth_feeHistory` | NOT IMPLEMENTED | Returns a collection of historical block gas data |
+| `ETH` | `eth_gasPrice` | NOT IMPLEMENTED | Returns the current price per gas in wei |
+| `ETH` | `eth_getBalance` | NOT IMPLEMENTED | Returns the balance of the account of given address |
+| `ETH` | `eth_getBlockByHash` | NOT IMPLEMENTED | Returns information about a block by block hash |
+| `ETH` | `eth_getBlockByNumber` | NOT IMPLEMENTED | Returns information about a block by block number |
+| `ETH` | `eth_getBlockTransactionCountByHash` | NOT IMPLEMENTED | Number of transactions in a block from a block matching the given block hash |
+| `ETH` | `eth_getBlockTransactionCountByNumber` | NOT IMPLEMENTED | Number of transactions in a block from a block matching the given block number |
+| `ETH` | `eth_getCode` | NOT IMPLEMENTED | Returns code at a given address |
+| `ETH` | `eth_getCompilers` | NOT IMPLEMENTED | Returns a list of available compilers |
+| `ETH` | `eth_getFilterChanges` | NOT IMPLEMENTED | Polling method for a filter, which returns an array of logs, block hashes, or transaction hashes, depending on the filter type, which occurred since last poll |
+| `ETH` | `eth_getFilterLogs` | NOT IMPLEMENTED | Returns an array of all logs matching filter with given id |
+| `ETH` | `eth_getLogs` | NOT IMPLEMENTED | Returns an array of all logs matching a given filter object |
+| `ETH` | `eth_getProof` | NOT IMPLEMENTED | Returns the details for the account at the specified address and block number, the account's Merkle proof, and the storage values for the specified storage keys with their Merkle-proofs |
+| `ETH` | `eth_getStorageAt` | NOT IMPLEMENTED | Returns the value from a storage position at a given address |
+| `ETH` | `eth_getTransactionByBlockHashAndIndex` | NOT IMPLEMENTED | Returns information about a transaction by block hash and transaction index position |
+| `ETH` | `eth_getTransactionByBlockNumberAndIndex` | NOT IMPLEMENTED | Returns information about a transaction by block number and transaction index position |
+| `ETH` | `eth_getTransactionByHash` | NOT IMPLEMENTED | Returns the information about a transaction requested by transaction hash |
+| `ETH` | `eth_getTransactionCount` | NOT IMPLEMENTED | Returns the number of transactions sent from an address |
```

Comment:
This is `IMPLEMENTED`: https://github.com/matter-labs/era-test-node/blob/main/src/node.rs#L585